wazo_calld/plugins/conferences/bus_consume.py

```diff
@@ -104,6 +105,7 @@ def _notify_participant_unmuted(self, event):
             'language': event['Language'],
             'call_id': event['Uniqueid'],
         }
+        raw_participant['user_uuid'] = event.get('ChanVariable', {}).get('XIVO_USERUUID')
```

Comment:
```suggestion
            'call_id': event['Uniqueid'],
            'user_uuid': event.get('ChanVariable', {}).get('XIVO_USERUUID'),
        }
```
And for all other same changes

Reply:
Fixed, thank you